### ISSUE TITLE
Electron version affected by CVE 2018-15685

### ISSUE THREAD

**Issue by D4N** [UltimateHackingKeyboard/agent#874]
The version of Electron.js used by the agent appears to be affected by CVE 2018-15685: https://www.npmjs.com/advisories/732.

Can you please upgrade it to one of the unaffected versions (e.g. 1.8.8)?

**Comment by mondalaci** [comment]
We could update to new versions of various affected packages on almost a daily basis, and quite often some APIs break, so we don't have bandwidth for this. We update our dependencies in batches. Also, Agent doesn't do any network communication, so this doesn't seem serious.

**Comment by D4N** [comment]
Thanks for your reply. I can understand that, as the nodejs ecosystem is moving rather rapidly. 

I cannot judge whether the agent can be exploited via this vulnerability. However, I have tried to build the agent with electron 1.8.8, it passed all tests and it ran without any (at least to me) apparent bugs (but I have only remapped a few keys and didn't try anything more elaborate). So upgrading could be safe.

Also: thanks for creating the UHK! That thing is pretty damn awesome!
